sigs.yml

```
@@ -659,4 +659,22 @@
         github: reyang
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
-        github: jpkrohling
\ No newline at end of file
+        github: jpkrohling
+  - name: Project Tooling and Infrastructure
+    meeting: Weekly on Wednesday at 08:00 AM PT
+    notes:
+      type: gDoc
+      value: 1_dTP2XIaZoFjNiipkeMaGJN10NrhS20FiBTTJoQJeVM
+    chat:
+      - type: slack
+        name: '#otel-project-tooling'
+        id: C07BPU981PV
+    invites: calendar-project-tooling
+    sponsors:
+      - name: 'Austin Parker'
+        github: austinlparker
+      - name: 'Trask Stalnaker'
+        github: trask
+    gcLiaison:
+      - name: 'Austin Parker'
+        github: austinlparker
```

```suggestion
        github: austinlparker

```
so we have a clean diff next time